<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewClientModal</class>
 <widget class="QDialog" name="NewClientModal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Client</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="chromiumButton">
       <property name="text">
        <string>Chromium</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="chromeButton">
       <property name="text">
        <string>Chrome</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="firefoxButton">
       <property name="text">
        <string>Firefox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="anythingButton">
       <property name="text">
        <string>Anything (Port 8082)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="descLabel">
       <property name="text">
        <string>Launch a new Chromium instance:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>388</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="launchButton">
       <property name="text">
        <string>Launch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
